static/js/unacked.js

```diff
@@ -0,0 +1,33 @@
+import * as Table from './table.js'
+import { UrlDataSource } from './datasource.js'
+
+const search = new URLSearchParams(window.location.hash.substring(1))
+const queue = search.get('name')
+const vhost = search.get('vhost')
+const url = `api/queues/${encodeURIComponent(vhost)}/${encodeURIComponent(queue)}/unacked`
+const channelBaseUrl = 'channel#name='
+document.getElementById('queue-link').href = `/queue#vhost=${encodeURIComponent(vhost)}&name=${encodeURIComponent(queue)}`
```

Comment:
let it work when /api is submapped into another path, so don't use absolute urls. 

```suggestion
document.getElementById('queue-link').href = `queue#vhost=${encodeURIComponent(vhost)}&name=${encodeURIComponent(queue)}`
```